Playbooks: Terminology generalization (#4716)
mattermost/docs: source/incident-collaboration/setting-up-playbooks.rst

Planning a Playbook
====================

Playbooks are made up of:

- **Checklists:** The list of tasks to be completed for the run.
- **Templates:** Templates for frequently-used actions such as updates and reminders. 
- **Actions:** Automation options for inviting members, webhooks, welcome messages and more.
- **Permissions:** Manage permissions at a channel and a playbook level.

Creating checklists
-------------------

1. Go to **Main Menu > Incident Collaboration**.
2. Select **Playbooks**.
3. Start a **Blank Playbook**, or use the built-in template.
4. Name your playbook and provide a description.
5. Select the **Checklists** tab.

> You can start with the default checklist and edit it or you can delete it and select **+ New checklist**. Each new section of a checklist is called a stage.
> 
>     * Within each stage, select **+ New task** to add tasks that are meant to be completed together.
>     * Drag and drop to reorganize stages and tasks.

Reviewer: Checklists here?

```suggestion
You can start with the default checklist and edit it or you can delete it and select **+ New checklist**.

    * Within each checklist, select **+ New task** to add tasks that are meant to be completed together.
    * Drag and drop to reorganize checklists and tasks.
```

Defining actions
----------------

You can customize actions associated with your playbook by setting up keyword triggers, automate member invites, and add a welcome message for new members.

Select the **Actions** tab.

Prompt to run the playbook when a user posts a message
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  * **Keywords**: Enable the toggle, and enter comma-separated keywords that, when included in a message, will prompt the posting user to run the playbook.

.. image:: ../images/Playbook-keyword-monitoring.png
   :alt: Keyword notification.

When a run starts
~~~~~~~~~~~~~~~~~

  * **Invite members**: Enable the toggle, and select members who will automatically be invited to the channel when the playbook run starts.
  * **Assign the owner role**: Enable the toggle and select a member. This member is automatically assigned as the owner of the playbook run.
  * **Announce in another channel**: Enable the toggle and select a channel. When the playbook run is started, an announcement is made in the selected channel.

>   * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for status updates.

Reviewer: What about something like this?

```suggestion
  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts.
```
